apps/meteor/packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -3088,6 +3088,8 @@
   "List_of_Direct_Messages": "List of Direct Messages",
   "List_view": "List View",
   "Livechat": "Livechat",
+  "Livechat_AdditionalWidgetScripts": "Livechat Widget Additional Scripts. Its possible to specify a CSV list of scripts, for example : https://yourUrl/customScrip1.js,https://yourUrl/customScrip2.js",
```

Comment:
```suggestion
"Livechat_additional_widget_scripts": "Livechat Widget Additional Scripts"
"Livechat_additional_widget_scripts_Description": "Its possible to specify a CSV list of scripts, for example : https://yourUrl/customScrip1.js,https://yourUrl/customScrip2.js"
```

Reply:
We should use a pair of title & descriptions on the settings, this suggestion applies to the classes as well